index.html

```diff
@@ -148,28 +148,29 @@ <h2 class="pt-3" id="info">What <em>is</em> Standard?</h2>
           href="https://magic.wizards.com/en/content/standard-formats-magic-gathering" rel="noopener">
             <b>Standard</b>
           </a>
-          is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're released and
-          drop out about twenty-one months later.
+          is a tournament format which consists of a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter
+          the format when they're released and drop out about twenty-one months later.
           </p>
           <p>
-          Generally Standard will grow until it would receive its ninth set, then instead it drops its oldest four sets
-          to receive it as a fifth set. There are exceptions to this. It's confusing.
+          Generally it grows until it contains eight sets. Then — with the next new joining release — the oldest four sets
+          are dropped as group and the new one is added as fifth legal set.<br>
+          There are exceptions to this. It's confusing.
```

Comment:
>What's the reason for the `<br>`? The last sentence seems like it belongs in the same paragraph.

👍 

>This explanation was definitely confusing before, but I feel like it's even more confusing now just because it's less succinct. I'm totally down if you want to reinvent this sentence from scratch though, instead of carrying my baggage with your edits.

Gave it a new try and combined the paragraphs as well. Hope this improves the explanation without too much word repetition.